java/com/google/javascript/jscomp/JsCheckerPassConfig.java

```diff
@@ -111,13 +110,6 @@ private PassFactory scopedAliases() {
         .build();
   }
 
-  private PassFactory closureRewriteClass() {
-    return PassFactory.builder()
-        .setName("closureRewriteClass")
-        .setInternalFactory((compiler) -> new ClosureRewriteClass(compiler))
```

Comment:
`ClosureRewriteClass` has since been deleted. It was used to support `goog.defineClass`.